lib/fluent/supervisor.rb

```diff
@@ -134,13 +137,25 @@ def run_rpc_server
 
       @rpc_server.mount_proc('/api/config.gracefulReload') { |req, res|
         $log.debug "fluentd RPC got /api/config.gracefulReload request"
-        if Fluent.windows?
-          supervisor_sigusr2_handler
+        if @block_reload_until > Fluent::Clock.now
+          $log.warn "block grcefulReload until: #{Time.now + @block_reload_until - Fluent::Clock.now} " +
+                    "remaining: #{(@block_reload_until - Fluent::Clock.now).to_i} seconds"
+          [422, nil, {'message': 'failed to reload config'}]
         else
-          Process.kill :USR2, $$
-        end
+          if config[:blocking_reload_interval]
```

Comment:
Is this check needed?
`@blocking_reload_interval` cannot be used here?

Reply:
Ah, `@blocking_reload_interval` should be used here.
Then, it seems that it is a redundant conditional assignment.

Thanks!